app/assets/javascripts/z.js

```diff
@@ -8,6 +8,15 @@ $(document).ready(function(){
       $("#overlay-modal").show(); }
     });
 
+  $("#autocomplete-tv-series-search-submit").autocomplete({
+    source: $("#autocomplete-tv-series-search-submit").data("autocomplete-source"),
+    minLength: 4,
+    select: function(event, ui) {
+      $("#autocomplete-tv-series-search-submit").val(ui.item.value);
+      $(this).closest("form").submit();
+      $("#overlay-modal").show(); }
+    });
+
```

Comment:
😓 I was hacking various things trying to get this to work without adding a new function, and the following hack (I mean, everything in jquery feels like a hack) seems to work:

```js
  $('.autocomplete-auto-submit').each(function(){
    let form = $(this)
    let source = form.data('autocomplete-source')

    form.autocomplete({
      source: source,
      minLength: 4,
      select: function (event, ui) {
        $(".autocomplete-auto-submit").val(ui.item.value);
        $(this).closest("form").submit();
        $("#overlay-modal").show();
      }
    })
  });
```

Reply:
🎉 😮 Nice! This totally works for both movies and tv series. There is a brief moment right after you select the item from the dropdown and before the submit action kicks in that the selected title is populated in both of these search fields. 
<img width="739" alt="Screen Shot 2021-12-20 at 7 29 32 PM" src="https://user-images.githubusercontent.com/8680712/146855559-159f6d60-fe7a-4f07-831a-2751c20e706e.png">
How do you feel about that functionality vs duplicating a function in js?

Reply:
Actually, this fixes it:
```javascript
// replace 
$(".autocomplete-auto-submit").val(ui.item.value);

// with this
$(this).closest(".autocomplete-auto-submit").val(ui.item.value);
```